src/components/prescriptions/PrescriptionAddComponent.vue

```diff
@@ -64,10 +64,18 @@
                     wide
                     >Zapisz receptę</button-component
                 >
-                <bouncing-balls-component :visible="isLoading" />
-                <error-info :message="error" />
+                <bouncing-balls-component
+                    role="presentation"
+                    alt="loader"
+                    aria-label="loader"
+                    :visible="isLoading" />
+                <error-info
+                    alt="error value"
+                    aria-label="error value"
+                    :message="error" />
                 <span
                     v-show="successFlag"
+                    aria-live="prescription send"
```

Comment:
```suggestion
                    aria-live="Recepta została przesłana pomyślnie"
```

Reply:
aria-live is not for adding message. it is used to indicate in which moment SR should read. There are few params such as polite. you have to also add aria-label